config/config.sample.yaml

```diff
@@ -102,3 +102,9 @@ team_sync:
 
 provisioning:
   enabled: true
+  # Should the bridge allow public rooms to be bridged
+  require_public_room: true
```

Comment:
The comment says "should?", the variable name says "require".
This is confusing.